src/openforms/conf/base.py

```diff
@@ -1270,6 +1270,7 @@
 # DJANGO-UPGRADE-CHECK
 #
 UPGRADE_CHECK_PATHS: UpgradePaths = {
+    "3.3.0": UpgradeCheck(VersionRange(minimum="3.2.0")),
```

Comment:
The task says minimum required version should be `3.1.0`, why do we have `3.2.0` here?

Reply:
Typo in the ticket! Upgrade check was added in 3.2.0